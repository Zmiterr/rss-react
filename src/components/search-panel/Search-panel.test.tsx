import React from "react";
import SearchPanel from './Search-panel';
//test deps
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter } from 'react-router-dom';



Enzyme.configure({ adapter: new Adapter() })


describe("SearchPanel component", () => {
	it("should render SearchPanel component", () => {
		const component = shallow(<BrowserRouter><SearchPanel  searchValue={''}/></BrowserRouter>);
		expect(component).toMatchSnapshot();
	});
});
