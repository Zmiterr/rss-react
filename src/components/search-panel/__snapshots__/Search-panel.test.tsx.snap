// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPanel component should render SearchPanel component 1`] = `ShallowWrapper {}`;
